### ISSUE TITLE
Export esm version of the package

### ISSUE THREAD

**Issue by idanen** [olivierwilkinson/webdriverio-testing-library#5]
Currently can't use this library in a `type="module"` package.
Can we provide an export for esm environments to be used like:
```
import {setupBrowser} from '@testing-library/webdriverio/esm'
```
?

**Comment by olivierwilkinson** [comment]
Hi there 😄 

I don't see why not! I'll look into it at some point this week 👍

**Comment by idanen** [comment]
Opened #6 for your review 😄

**Comment by olivierwilkinson** [comment]
> Opened #6 for your review 😄

~~Hi! We ended up working on it at the same time! My bad, it was definitely harder to get to this than I expected this last week. I've released at `@testing-library/webdriverio@2.1.0-prerelease.1` if you want to try it out and make sure it's working for you. If not I'll working with you on your PR to get that passing CI :)~~

Edit: I actually prefer your PR so I'll review it to get it passing the CI checks so we can go with that one 😄
